src/api/app.py

```diff
@@ -95,13 +102,19 @@ async def proxy(request: Request, path: str):
         headers["Authorization"] = f"Bearer {access_token}"
 
         try:
+            content = await request.body()
+
+            if USE_MODEL_MAPPING:
+                request_model = content.get("model", None)
+                content["model"] = get_model("GCP", request_model)
```

Comment:
where does GCP come from

Reply:
Should this map also handle the case for the managed APIs that have a model name in their URL? IIRC there was an API like that, but I forgot. @jordanstephens ?

Reply:
There is, but I've considered it out of scope for now. Let's stay focused on the primary foundation model offerings.